docs/content/getting-started/configure.md

```diff
@@ -1,52 +1,419 @@
+### `allow-missing-formatter`
+
+Do not exit with error if a configured formatter is missing.
+
+=== "Flag"
+
+    ```console
+    treefmt --allow-missing-formatter true
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_ALLOW_MISSING_FORMATTER=true treefmt
+    ```
+
+=== "Config"
+
+    ```toml
+    allow-missing-formatter = true
+    ```
+
+### `ci`
+
+Runs treefmt in a CI mode, enabling [no-cache](#no-cache), [fail-on-change](#fail-on-change) and adjusting some other settings best suited to a
+continuous integration environment.
+
+=== "Flag"
+
+    ```console
+    treefmt --ci
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_CI=true treefmt
+    ```
+
+### `clear-cache`
+
+Reset the evaluation cache. Use in case the cache is not precise enough.
+
+=== "Flag"
+
+    ```console
+    treefmt -c
+    treefmt --clear-cache
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_CLEAR_CACHE=true treefmt
+    ```
+
+### `config-file`
+
+=== "Flag"
+
+    ```console
+    treefmt --config-file /tmp/treefmt.toml
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_CONFIG=/tmp/treefmt.toml treefmt
+    ```
+
+### `cpu-profile`
+
+The file into which a [pprof](https://github.com/google/pprof) cpu profile will be written.
+
+=== "Flag"
+
+    ```console
+    treefmt --cpu-profile ./cpu.pprof
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_CPU_PROFILE=./cpu.pprof treefmt
+    ```
+
+=== "Config"
+
+    ```toml
+    cpu-profile = "./cpu.pprof"
+    ```
+
+### `excludes`
+
+An optional list of [glob patterns](#glob-patterns-format) used to exclude files from all formatters.
+
+=== "Flag"
+
+    ```console
+    treefmt --excludes *.toml,*.php,README
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_EXCLUDES="*.toml,*.php,README" treefmt
+    ```
+
+=== "Config"
+
+    ```toml
+    excludes = ["*.toml", "*.php", "README"]
+    ```
+
+### `fail-on-change`
+
+Exit with error if any changes were made during execution.
+
+=== "Flag"
+
+    ```console
+    treefmt --fail-on-change true
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_FAIL_ON_CHANGE=true treefmt
+    ```
+
+=== "Config"
+
+    ```toml
+    fail-on-change = true
+    ```
+
+### `formatters`
+
+A list of formatters to apply.
+Defaults to all configured formatters.
+
+=== "Flag"
+
+    ```console
+    treefmt -f go,toml,haskell
+    treefmt --formatters go,toml,haskell
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_FORMATTERS=go,toml,haskell treefmt
+    ```
+
+=== "Config"
+
+    ```toml
+    formatters = ["go", "toml", "haskell"]
+
+    [formatter.go]
+    ...
+
+    [formatter.toml]
+    ...
+
+    [formatter.haskell]
+    ...
+
+    [formatter.ruby]
+    ...
+
+    [formatter.shellcheck]
+    ...
+    ```
+
+### `no-cache`
+
+Ignore the evaluation cache entirely. Useful for CI.
+
+=== "Flag"
+
+    ```console
+    treefmt --no-cache
+    ```
+
+=== "Env"
+
+    ```console
+    TREEFMT_NO_CACHE=true treefmt
+    ```
+
+### `on-unmatched`
```

Comment:
Should we mention here that this can affect the exit code of treefmt? IIRC, `treefmt --on-unmatched=fatal` will exit non-zero if there are fatal logs.

Reply:
Yeah, I'll create a follow up PR.